views/standard/spec/columns.js

```diff
@@ -229,7 +229,7 @@ define(['jquery', 'appearanceUtils', 'underscore'], function($, utils, _) {
          id: "qaPulls",
          selector: function(pull) {
             return !pull.qa_done() && !pull.dev_blocked() &&
-             pull.build_succeeded();
+             pull.passing;
```

Comment:
Any particular reason for using `passing` rather than `isPassing()`? It seems safer to always use a computed value.

Reply:
This is on the frontend, where we don't have access to the config. ideally, yes if we didn't have this constraint, I would have preferred it to do it the way you suggested.

Reply:
Makes sense. I didn't realize this ran on the frontend.